pkg/messagebroker/kafka.go

```diff
@@ -130,6 +132,14 @@ func newKafkaProducerClient(ctx context.Context, bConfig *BrokerConfig, options
 		return nil, err
 	}
 
+	go func() {
+		logger.Ctx(ctx).Infow("starting producer log reader...")
+		select {
+		case log := <-p.Logs():
+			logger.Ctx(ctx).Infow("kafka producer logs", "options", options, "log", log.String())
```

Comment:
do we need to log options with every request

Reply:
added this for my debugging purpose. Not up for review.